x-pack/plugins/observability_solution/observability/server/lib/annotations/index_templates/annotation_index_templates.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const ANNOTATION_RESOURCES_VERSION = 1.3;
+
+export const ANNOTATION_INDEX_TEMPLATE_NAME = 'observability-annotations';
+export const ANNOTATION_INDEX_TEMPLATE_PATTERN = `observability-annotation*`;
+
+export const ANNOTATION_COMPONENT_TEMPLATE_MAPPINGS_NAME = 'observability-annotations-mappings';
+export const ANNOTATION_COMPONENT_TEMPLATE_SETTINGS_NAME = 'observability-annotations-settings';
+
+export const getAnnotationIndexTemplate = (index: string) => ({
+  name: ANNOTATION_INDEX_TEMPLATE_NAME,
```

Comment:
Does this remove the ability now for users to specify their own index name? I don't really see the benefit of that feature, but it's part of what we support currently, isn't it?